libs/libc/stdio/lib_libvsprintf.c

```diff
@@ -488,6 +488,61 @@ static int vsprintf_internal(FAR struct lib_outstream_s *stream,
 
       if (c == 'p')
         {
+          unsigned char sub_c = fmt_char(fmt);
+
+          switch (sub_c)
+            {
+              case 'V':
+                {
+                  FAR struct va_format *vaf = va_arg(ap, void *);
```

Comment:
```suggestion
                  FAR struct va_format *vaf = va_arg(ap, FAR struct va_format *);
```
But I'm not sure if that is possible